grc/core/generator/top_block.py

```diff
@@ -336,16 +336,13 @@ def by_domain_and_blocks(c):
                 # the code for each subconnection
                 porta = con.source_port
                 portb = con.sink_port
-                fg = self._flow_graph
 
                 if porta.dtype == 'bus' and portb.dtype == 'bus':
                     # which bus port is this relative to the bus structure
                     if len(porta.bus_structure) == len(portb.bus_structure):
                         for port_num_a, port_num_b in zip(porta.bus_structure, portb.bus_structure):
                             hidden_porta = porta.parent.sources[port_num_a]
                             hidden_portb = portb.parent.sinks[port_num_b]
-                            connection = fg.parent_platform.Connection(
-                                parent=self, source=hidden_porta, sink=hidden_portb)
```

Comment:
If you look elsewhere in this file, where `connection_factory()` is used, `self` is *not* passed in as a parent. This only happens here. However, since this wasn't used anyway, I decided to just remove it.